bsip-0083.md

```diff
@@ -0,0 +1,74 @@
+```
+BSIP: 0083
+Title: Decouple BitAssets from Platform Governance Process
+Authors:    
+   Alfredo Garcia
+   Christopher Sanborn
+   John M. Jones
+   Michel Santos
+   Nathan Hourt
+   Peter Conrad
+   Ryan R. Fox
+Status: Draft
+Type: Protocol
+Obsoletes: BSIP-0016, BSIP-0058, BSIP-0059, BSIP-0075, BSIP-0076
+Created: 2019-10-07
+Discussion: https://github.com/bitshares/bsips/issues/239
+Worker: <TBD>
+```
+
+# Abstract
+This BSIP separates the governance of the technical security of the blockchain from the governance of BitAssets.  BitAssets are a collection of special case SmartCoin which conflate blockchain governance and market effectiveness. To best serve the BitShares protocol, the existing set of BitAssets must be reconfigured to separate the concerns of block production, price feeds, and market operations; that is to say: deterministic consensus, non-deterministic external data, and smart contract settings.
```

Comment:
The abstract does not mention the primary focus of this BSIP which is requesting an asset ownership transfer from the committee-account to the cn-vote account.

What are the 'must have' bitasset reconfigurations which the committee is unable to perform?

Is the committee unable to achieve deterministic consensus on bitasset settings? What proof is there for this?

Why can't non-deterministic external data be a pull request for witness price feeds instead of requesting asset ownership transfer?